Mage.Sets/src/mage/cards/m/MarduOutrider.java

```diff
@@ -0,0 +1,38 @@
+package mage.cards.m;
+
+import mage.MageInt;
+import mage.abilities.Ability; // not sure about this line - think it is not needed
```

Comment:
fyi - Unused imports are harmless, though I certainly recommend just having your IDE manage that sort of thing automatically